src/Explorer/Controls/NotebookGallery/GalleryViewerComponent.tsx

```diff
@@ -406,8 +404,9 @@ export class GalleryViewerComponent extends React.Component<GalleryViewerCompone
     const props: GalleryCardComponentProps = {
       data,
       isFavorite,
+      showDownload: !!this.props.container,
       showDelete: this.state.selectedTab === GalleryTab.Published,
-      onClick: () => this.openNotebook(data, isFavorite),
+      onClick: () => this.props.openNotebook(data, isFavorite),
```

Comment:
FYI, there's a `AccessibleElement` component in `Controls/` that helps make clickable elements accessible by catching Enter/Space key presses and requiring aria-label.

Reply:
I just noticed that you use `<FocusZone>` for the cards, so you can prob ignore my previous comment.